File: app/src/main/java/com/antonio/samir/meteoritelandingsspots/ui/meteoriteRecyclerView/selector/MeteoriteSelectorLandscape.java
package com.antonio.samir.meteoritelandingsspots.ui.meteoriteRecyclerView.selector;


import com.antonio.samir.meteoritelandingsspots.model.Meteorite;

public class MeteoriteSelectorLandscape implements MeteoriteSelector {

    private final MeteoriteSelectorView view;

    public MeteoriteSelectorLandscape(MeteoriteSelectorView view) {
        this.view = view;
    }

    @Override
    public void selectItemId(final Meteorite meteorite) {
        view.selectLandscape(meteorite);
    }

}

File: app/src/main/java/com/antonio/samir/meteoritelandingsspots/features/list/ui/recyclerView/selector/MeteoriteSelectorLandscape.kt
package com.antonio.samir.meteoritelandingsspots.features.list.ui.recyclerView.selector

import com.antonio.samir.meteoritelandingsspots.data.repository.model.Meteorite


class MeteoriteSelectorLandscape(private val view: MeteoriteSelectorView) : MeteoriteSelector {

    override fun selectItem(meteorite: Meteorite) {
        view.selectLandscape(meteorite)
    }

}
